hbt/config/triggers.py

```diff
@@ -1539,13 +1548,450 @@ def add_triggers_2024(config: od.Config) -> None:
+                # filter names:
+                # hltHpsOverlapFilterIsoMu20LooseMuTauWPPNetPFJet27 (OverlapFilter PFTau)
+                trigger_bits=get_bit_sum_v("mu", [
+                    "OverlapFilterPFTau",
+                    "MuTau",
+                ]),
+            ),
+            tau=TriggerLeg(
+                pdg_id=15,
+                # min_pt=None,  # cut on reco objects, not TrigObj
+                # filter names:
+                # hltHpsOverlapFilterIsoMu20LooseMuTauWPPNetPFJet27 (PNet + HPS)
+                trigger_bits=get_bit_sum_v("tau", [
+                    "PNet",
+                    "HPS",
+                    "OverlapFilterIsoMu",
+                    "MuTauFilter" if nano_version == 15 else None,
+                    "MatchL1HLT",
+                ]),
+            ),
+        ),
+        applies_to_dataset=(lambda dataset_inst: dataset_inst.is_mc or dataset_inst.has_tag("mutau")),
+        tags={"cross_trigger", "cross_mu_tau"},
+    )
+     #
+    # tauh tauh pnet
+    #
+    config.x.triggers.add(
+        name="HLT_DoublePNetTauhPFJet30_Medium_L2NN_eta2p3",
+        id=510,
+        legs=dict(
+            tau1=TriggerLeg(
+                pdg_id=15,  
+                # min_pt=None,  # cut on reco objects, not TrigObj
+                # filter names:
+                # hltDoublePFJets30PNetTauhTagMediumWPL2DoubleTau (PNet + HPS)
+                trigger_bits=get_bit_sum_v("tau", [
+                    "PNetDiTau",
+                    "HPS",
+                    "Medium" if nano_version == 15 else None,
+                ]),
+            ),
+            tau2=TriggerLeg(
+                pdg_id=15,
+                # min_pt=None,  # cut on reco objects, not TrigObj
+                # filter names:
+                # hltDoublePFJets30PNetTauhTagMediumWPL2DoubleTau (PNet + HPS)
+                trigger_bits=get_bit_sum_v("tau", [
+                    "PNetDiTau",
+                    "HPS",
+                    "Medium" if nano_version == 15 else None,
+                ]),
+            ),
+        ),
+        applies_to_dataset=(lambda dataset_inst: dataset_inst.is_mc or dataset_inst.has_tag("tautau")),
+        tags={"cross_trigger", "cross_tau_tau"},
+    )
+ #
+    # tau tau jet pnet
+    #
+    config.x.triggers.add(
+        name="HLT_DoublePNetTauhPFJet26_L2NN_eta2p3_PFJet60",
+        id=701,
+        legs=dict(
+            tau1=TriggerLeg(
+                pdg_id=15,
+                # min_pt=None,  # cut on reco objects, not TrigObj
+                # filter names:
+                # hltHpsOverlapFilterDoublePNetTauh26PFJet60
+                trigger_bits=get_bit_sum_v("tau", [
+                    "PNet",
+                    "DiTauAndPFJet",
+                ]),
+            ),
+            tau2=TriggerLeg(
+                pdg_id=15,
+                # min_pt=None,  # cut on reco objects, not TrigObj
+                # filter names:
+                # hltHpsOverlapFilterDoublePNetTauh26PFJet60
```

Comment:
```suggestion
                # hltDoublePFJets26PNetTauhTagL2DoubleTauJet
```

Not the last filter overall but the only one that gets matched by the filter bits and the last from the tau legs